sdk/communication/azure-communication-chat/tests/test_chat_thread_client_e2e.py

```diff
@@ -217,17 +236,18 @@ def test_send_read_receipt(self):
     @pytest.mark.live_test_only
     def test_list_read_receipts(self):
         self._create_thread()
-        self._send_message()
 
-        # send read receipts first
-        self.chat_thread_client.send_read_receipt(self.message_id)
-        if self.is_live:
-            time.sleep(2)
+        # send messages and read receipts
+        for i in range(2):
+            self._send_message()
+            self.chat_thread_client.send_read_receipt(self.message_id)
+            if self.is_live:
+                time.sleep(2)
```

Comment:
here too. let's try to avoid `time.sleep()`

Reply:
The time.sleep() actions have been isolated inside a helper function now. Confirmed the approach with @annatisch as well. Without the sleeps(), the e2e functionality is not tested properly.